ray-operator/DEVELOPMENT.md

```diff
@@ -79,6 +79,22 @@ kubectl logs {YOUR_OPERATOR_POD} | grep "Hello KubeRay"
 * The command `make docker-build` (Step 3) will also run `make test` (unit tests).
 * Step 6 also installs the custom resource definitions (CRDs) used by the KubeRay operator.
 
+### Start kuberay-operator locally
```

Comment:
We do not encourage contributors to develop KubeRay outside of the k8s cluster.

Reply:
Got it. I removed the instruction.